server/src/main/java/io/deephaven/server/table/ops/AggregationAdapter.java

```diff
@@ -79,6 +81,12 @@ public static Count adapt(AggregationCount count) {
         return Aggregation.AggCount(count.getColumnName());
     }
 
+    public static Formula adapt(AggregationFormula formula) {
+        // TODO: until the deprecated AggFormula() that uses paramToken is removed, we must re-create this as a
+        // combined string
+        return Aggregation.AggFormula(formula.getColumnName() + "=" + formula.getFormula());
```

Comment:
Why? You can do `Formula.of(formula.getColumnName(), formula.getFormula())` I think?

Reply:
Oversight on my part, I introduced the 2 parameter variant specifically for this case :(